source/openvino.js

```diff
@@ -126,7 +126,16 @@ openvino.Graph = class {
         this._arguments = {};
 
         for (const layer of this._const(net.layers, net.edges)) {
-            const inputs = layer.inputs.map((input) => this._argument(layer.id, layer.precision, input, net.edges));
+            const inputs = layer.inputs.map((input) => {
+                const to = layer.id + ':' + input.id;
+                const [outputLayerId, outputId] = net.edges[to].split(':');
+                const outputLayer = net.layers.find((layer) => layer.id === outputLayerId);
+                if (outputLayer) {
+                    const output = outputLayer.outputs.find((output) => output.id === outputId);
+                    input.precision = output.precision;
```

Comment:
* What exactly is this code doing?
* Can the `precision` of the output always assumed to be the same as the input.
* `input` contains source data that should be copied instead of modified.

Reply:
Initially in XML file inputs don't have precision property, unlike outputs. So, I try to match inputs and outputs according to the edges map and assign precisions to inputs.
Yes, precision to the output always assumed to be the same as the input.

Reply:
@Vladimir9812 "assumed" to be the same as in this code makes this assumption but it could be wrong or is this always the case for all OpenVINO models?

Reply:
@lutzroeder, sorry for my long answers...Yes, it is true for all OpenVINO models.
Thank you for merge. Can I find a smaller int4 model and attach it to this PR?

Reply:
Yes, that would be great.

Reply:
[simple_model_int4.zip](https://github.com/lutzroeder/netron/files/6278507/simple_model_int4.zip)